spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe "the guests show page" do
+  let!(:guest_1) { Guest.create!(name: "Andi", nights: 2) }
+  let!(:hotel_1) { Hotel.create!(name: "Sleepy Inn", location: "Colorado") }
+  let!(:room_1) { hotel_1.rooms.create!(rate: 125, suite: "Presidential") }
+  let!(:room_2) { hotel_1.rooms.create!(rate: 110, suite: "King") }
+  let!(:room_3) { hotel_1.rooms.create!(rate: 170, suite: "Luxury") }
+  let!(:guest_room_1) { GuestRoom.create!(guest_id: guest_1.id, room_id: room_1.id) }
+  let!(:guest_room_2) { GuestRoom.create!(guest_id: guest_1.id, room_id: room_2.id) }
+
+  before do
+    visit "/guests/#{guest_1.id}"
+  end
+
+
+  it "shows guest name and list of rooms and room attributes with hotel name" do
+    expect(page).to have_content("Guest Name: #{guest_1.name}")
```

Comment:
within blocks could be helpful here to break up these assertions and ensure they're grouped together properly